Cargo.toml

```diff
@@ -21,8 +21,8 @@ lazy_static = "1.4.0"
 itertools = "0.9.0"
 # fixed = "1.9.0"
 clap = "3.0.0-beta.2"
-z3 = {version="0.10.0", features = ["static-link-z3"]}
-# z3 = {version="0.10.0"}
+#  z3 = {version="0.10.0", features = ["static-link-z3"]}
+z3 = {version="0.10.0"}
```

Comment:
Ooof, no. Reverting this. Honestly, this is the worst part of the build process. The build time with the static-link is super long, but its unreasonable to assume people have `libz3` installed on their system.

Reply:
ugh yeah sorry about this. It's annoying for sure.